src/lib/com/yetanalytics/xapipe/filter/concept.clj

```diff
@@ -57,57 +44,58 @@
                          ::contextCategoryActivityType
                          ::contextOtherActivityType
                          ::attachmentUsageType]))
-  :ret ::validator)
+  :ret ::t/template)
 
 (defn t-val
```

Comment:
this name was supposed to mean template-validator. Maybe a rename to make-template or something?

Reply:
That's funny, I thought of it like `template-value` so I just left it

Reply:
Agree with Cliff. You see, unlike C, Clojure is not a Spartan language :)

Reply:
Sure, I'll defer to the original author. `make-template` it is

Reply:
https://github.com/yetanalytics/xapipe/pull/68/commits/a3af45434926957c3ebe80fc4e0052e7fdec5af6